.github/workflows/main.yml

```
@@ -111,6 +111,7 @@ jobs:
     - integration-test
     - test
     - test_cygwin
+    - check_changed_folders
```

How about keeping the order alphabetic?
```suggestion
    - check_changed_folders
    - integration-test
    - test
    - test_cygwin
```